ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1257,11 +1241,14 @@ func (r *RayClusterReconciler) getHeadServiceIPAndName(ctx context.Context, inst
 		return "", "", fmt.Errorf("head service IP is empty. cluster name %s, filter labels %v", instance.Name, common.RayClusterHeadServiceListOptions(instance))
 	} else if runtimeServices.Items[0].Spec.ClusterIP == corev1.ClusterIPNone {
 		// We return Head Pod IP if the Head service is headless.
-		ip, err := r.getHeadPodIP(ctx, instance)
+		head, err := common.GetRayClusterHeadPod(ctx, r, instance)
```

Comment:
maybe use `headPod` instead to keep the naming consistent with the codebase.

Reply:
Thanks for catching all these.